systems/common/rest/client/subscriber/subscriber.go

```diff
@@ -20,7 +20,7 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-const SubscriberEndpoint = "/v1/subscribers"
+const SubscriberEndpoint = "/v1/subscriber"
```

Comment:
Why are we changing this?

Reply:
it should be `subscriber`, that was a typo.

Reply:
Normally speaking, rest resources names inside endpoints should be plural.  We need to update this in subscriber system api-gtw